homeassistant/components/lock/zwave.py

```diff
@@ -14,26 +14,72 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_NOTIFICATION = 'notification'
-
+ATTR_LOCK_STATUS = 'lock_status'
 LOCK_NOTIFICATION = {
     1: 'Manual Lock',
     2: 'Manual Unlock',
     3: 'RF Lock',
     4: 'RF Unlock',
     5: 'Keypad Lock',
     6: 'Keypad Unlock',
+    11: 'Lock Jammed',
     254: 'Unknown Event'
 }
 
+LOCK_ALARM_TYPE = {
+    9: 'Deadbolt Jammed',
+    18: 'Locked with Keypad by user',
```

Comment:
This is picky, but can you be consistent with the spaces after `user`?
"Locked with Keypad by user2"  vs  "Unlocked with Keypad by user 2"

I setup an automation based on this status message and had to go back and correct it because the naming wasn't consistent.